src/App/App.tsx

```diff
@@ -20,7 +20,7 @@ interface IState {
 }
 
 class App extends Component<IProps, IState> {
-	constructor(props: any) {
+	constructor(props: IProps) {
```

Comment:
This is an important change! We already created our IProps, so this was half done. But typing App props is pretty critical in a project focused on learning TypeScript 😅